src/archivematica/dashboard/components/accounts/backends.py

```diff
@@ -44,7 +48,35 @@ class CustomOIDCBackend(OIDCAuthenticationBackend):
     Provide OpenID Connect authentication
     """
 
-    def get_settings(self, attr, *args):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        # Store additional settings as instance attributes.
+        self.OIDC_OP_SET_ROLES_FROM_CLAIMS = getattr(
+            settings, "OIDC_OP_SET_ROLES_FROM_CLAIMS", False
+        )
+
+        self.OIDC_OP_ROLE_CLAIM_PATH = getattr(
+            settings, "OIDC_OP_ROLE_CLAIM_PATH", "realm_access.roles"
+        )
+
+        self.OIDC_ACCESS_ATTRIBUTE_MAP = getattr(
+            settings, "OIDC_ACCESS_ATTRIBUTE_MAP", settings.DEFAULT_OIDC_CLAIMS
+        )
+
+        # Valid user roles which may be extracted from OIDC token.
+        self.USER_ROLE_ADMIN = getattr(settings, "USER_ROLE_ADMIN", "admin")
+        self.USER_ROLE_DEFAULT = getattr(settings, "USER_ROLE_DEFAULT", "default")
```

Comment:
Is there a way for the user to override these values from the `settings` modules?

Reply:
There should be in case the role name coming in the token cannot be changed. I will fix this.